agent/tool-scripts/bpftrace

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
+# The following script runs a bpftrace program with the program supplied as argument in pbench-register-tool
+# pbench-register-tool --name=bpftrace -- --bpf_prog=sample_prog.bt
+
+script_path=`dirname $0`
+script_name=`basename $0`
+pbench_bin="`cd ${script_path}/..; /bin/pwd`"
+
+# source the base script
+. "$pbench_bin"/base
+
+# Perftool scripts must provide the following functions
+# 1) Install the tool
+# 2) Start data collection
+# 3) Stop data collection
+# 4) post-process the data
+
+# Defaults
+tool=bpftrace
+tool_bin=/usr/local/bin/$tool
+group=default
+dir=""
+mode=""
+iteration="1"
+options="none"
+interval=5
+
+function usage() {
+	echo "
+	USAGE:
+	$script_name: you specified an invalid option
+	The following options are available:
+	--install,			          install this perf tool
+	--start|stop|postprocess	start/stop/post-process the data collection
+	--dir=str			            directory to store data collection (required)
+    --bpf_prog
```

Comment:
Seems like there is a tabs vs spaces problem with this file.  Can you be sure to use tabs and not spaces when indenting?